src/test/java/io/jenkins/plugins/analysis/core/model/ReferenceDetailsModelTest.java

```diff
@@ -0,0 +1,144 @@
+package io.jenkins.plugins.analysis.core.model;
+
+import java.util.List;
+import java.util.Locale;
+
+import org.apache.commons.text.StringEscapeUtils;
+import org.junit.jupiter.api.Test;
+
+import edu.hm.hafner.analysis.Issue;
+import edu.hm.hafner.analysis.IssueBuilder;
+import edu.hm.hafner.analysis.Report;
+import edu.hm.hafner.analysis.Severity;
+
+import io.jenkins.plugins.analysis.core.model.FileNameRenderer.BuildFolderFacade;
+import io.jenkins.plugins.analysis.core.model.StaticAnalysisLabelProvider.DefaultAgeBuilder;
+import io.jenkins.plugins.analysis.core.scm.BlameRequest;
+import io.jenkins.plugins.analysis.core.scm.Blames;
+
+import static j2html.TagCreator.*;
+import static org.assertj.core.api.Assertions.*;
+import static org.mockito.ArgumentMatchers.*;
+import static org.mockito.Mockito.*;
+
+/**
+ * Tests the class {@link ReferenceDetailsModel}.
+ *
+ * @author Colin Kaschel
+ */
+class ReferenceDetailsModelTest {
```

Comment:
Can you please create an abstract base class for `ReferenceDetailsModelTest` and `DetailsTableModelTest`? There you can move methods and properties that are used in both test cases. You can use overridable methods if something needs to be different.